common/src/main/java/com/wynntils/services/map/PoiService.java

```diff
@@ -75,87 +74,21 @@ public class PoiService extends Service {
 
     public PoiService() {
         super(List.of());
-
-        loadData();
     }
 
     @Override
-    public void reloadData() {
-        loadData();
+    public void registerDownloads(DownloadRegistry registry) {
+        registry.registerDownload(UrlId.DATA_STATIC_PLACES).handleReader(this::handlePlaces);
+        registry.registerDownload(UrlId.DATA_STATIC_SERVICES).handleReader(this::loadServices);
+        registry.registerDownload(UrlId.DATA_STATIC_COMBAT_LOCATIONS).handleReader(this::loadCombat);
+        registry.registerDownload(UrlId.DATA_STATIC_CAVE_INFO).handleReader(this::handleCaves);
```

Comment:
The naming is a bit inconsistent. Maybe rename all to "handle*"?

Reply:
Indeed. I think these were the only ones I missed though, I wanted all named handleX.